pkgs/top-level/all-packages.nix

```diff
@@ -12877,7 +12877,7 @@ with pkgs;
 
   qmapshack = libsForQt5.callPackage ../applications/gis/qmapshack { };
 
-  spatialite_gui = callPackage ../applications/gis/spatialite-gui {
+  spatialite-gui = callPackage ../by-name/sp/spatialite-gui/package.nix {
     inherit (darwin.apple_sdk.frameworks) Carbon Cocoa IOKit;
```

Comment:
```suggestion
```
probably no longer required

Reply:
Thank you.

Those packages are used [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/gis/spatialite-gui/default.nix#L67-L69). I can use community builder to test if package is still building, but I can't test if it really works correctly. I suggest to do this change in separate PR.

Reply:
If it builds, it works. Thanks to the recent SDK rework these inputs have been removed almost everywhere

Reply:
Thank you. I tested build using nix-community builder and it works fine.

```
$ nix-build --argstr system aarch64-darwin -A spatialite-gui

$ file ./result/bin/spatialite_gui
./result/bin/spatialite_gui: Mach-O 64-bit arm64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK|PIE>
```